spec/models/course/condition/assessment_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe Course::Condition::Assessment, type: :model do
+  it { is_expected.to act_as(:condition).class_name(Course::Condition.name) }
+
+  let(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+
+    describe '#title' do
+      context 'when there is no minimum grade percentage' do
+        it 'returns the correct title' do
+          subject.assessment = create(:course_assessment_assessment, title: 'Dummy', course: course)
+          expect(subject.title).to eq('Dummy')
+        end
+      end
+
+      context 'when there is minimum grade percentage' do
+        it 'returns the correct title' do
+          pending('Assessment#title uses number_to_percentage which uses I18 but we change I18
+behaviors in our test environment and this test will throw an exception. It works
+fine when the change is disabled.')
+          subject.assessment = create(:course_assessment_assessment, title: 'Dummy', course: course)
+          subject.minimum_grade_percentage = 33.333
+
+          expect(subject.title).to eq('Dummy (more than 33.33%)')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.